ccloud/docs/ccloud-demos-overview.rst

```diff
@@ -0,0 +1,189 @@
+=====================
+
+`Confluent Cloud <https://docs.confluent.io/current/cloud/index.html>`__ is a resilient, scalable streaming data service based on |ak|, delivered as a fully managed service. It has a web interface and local command line interface that you can use to manage cluster resources, Kafka topics, |sr|, and other services.
+
+This repo has a few resources to help you validate your solutions on |ccloud|.
+
+
+=======
+Warning
+=======
+
+All demos that connect to |ccloud| use real |ccloud| resources.
+To avoid unexpected charges, carefully evaluate the cost of resources before launching any demo and ensure all resources are destroyed after you are done running it.
+
+These demos are meant for development environments only, do not run any demo against your production |ccloud| cluster.
+These scripts create environments, clusters, topics, ACLs, service accounts, applications, and other resources.
+The scripts provide functions to delete created resources; however, you should verify the deletion of all demo resources manually.
+
+=====
+Demos
+=====
+
+Confluent Cloud Quickstart
+--------------------------
+
+The :devx-examples:`Confluent Cloud Quickstart|cp-quickstart/` is an automated version of the :ref:`Confluent Platform Quickstart <quickstart>`, but this one runs in |ccloud|.
+
+Fully Managed Stack in Confluent Cloud
+--------------------------------------
+
+The :devx-examples:`ccloud stack|ccloud/ccloud-stack/README.md` creates a stack of fully managed services in |ccloud|.
+Executed with a single command, it is a quick way to create fully managed components in |ccloud|, which you can then use for learning and building other demos.
+Please do not use this in a production environment.
+The script uses the |ccloud| CLI to dynamically do the following in |ccloud|:
+
+-  Create a new environment
+-  Create a new service account
+-  Create a new Kafka cluster and associated credentials
+-  Enable |sr-ccloud| and associated credentials
+-  Create a new KSQL app and associated credentials
+-  Create ACLs with wildcard for the service account
+-  Generate a local configuration file with all above connection information, useful for other demos/automation
+
+.. figure:: images/cloud-stack.png
+   :alt: image
+
+
+Client Code Examples
+--------------------
+
+If you are looking for code examples of producers writing to and consumers reading from |ccloud|, or producers and consumers using Avro with |sr-long|, refer to the :devx-examples:`client examples|clients/cloud/README.md`.
+It provides client examples written in various programming languages.
+
+.. figure:: ../clients/images/clients-all.png
+   :alt: image
+
+Confluent Cloud CLI
+-------------------
+
+The :devx-examples:`Confluent Cloud CLI demo|ccloud/beginner-cloud/README.md` is a fully scripted demo that shows users how to interact with |ccloud| using the |ccloud| CLI.
+It steps through the following workflow:
+
+-  Create a new environment and specify it as the default
```

Comment:
suggest: periods after bullets with complete sentences

Reply:
@tobydrake7 👍 and done